Sources/Mantis/CropView/CropView.swift

```diff
@@ -37,6 +37,9 @@ let hotAreaUnit: CGFloat = 32
 let cropViewPadding:CGFloat = 14.0
 
 class CropView: UIView {
+
+    public var dialConfig = Mantis.Config().dialConfig
```

Comment:
I don't think the property `dialConfig` here should be public because we don't expose it to users.